templates/base.html.twig

```diff
@@ -4,6 +4,7 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <link rel="icon" type="image/png" href="{{ asset('build/images/favicon.png') }}"/>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
```

Comment:
No CDN. You already had your bootstrap-icons dependancy in your node-modules, you should use it instead of this CDN. Look how you had bootstrap in your project, and do the same